src/layouts/sculpt.js

```diff
@@ -32,8 +43,26 @@ const Maker = (props) => {
           {maker.name}
         </Link>
       </div>
-      <h2 className="text-3xl my-6">
+      <h2 className="flex text-3xl my-6 justify-between">
```

Comment:
This is not good because the  `h2` is meant to be descriptive for a title not embedding a button. You need to do a placehold for the title and the button tho